Formula/c/cmctl.rb

```diff
@@ -21,17 +21,13 @@ class Cmctl < Formula
   def install
     ldflags = %W[
       -s -w
-      -X github.com/cert-manager/cert-manager/cmd/ctl/pkg/build.name=cmctl
-      -X github.com/cert-manager/cert-manager/cmd/ctl/pkg/build/commands.registerCompletion=true
+      -X github.com/cert-manager/cmctl/pkg/build.name=cmctl
+      -X github.com/cert-manager/cmctl/pkg/build/commands.registerCompletion=true
       -X github.com/cert-manager/cert-manager/pkg/util.AppVersion=v#{version}
       -X github.com/cert-manager/cert-manager/pkg/util.AppGitCommit=#{tap.user}
     ]
 
-    cd "cmd/ctl" do
-      system "go", "build", *std_go_args(ldflags:)
-    end
-
-    generate_completions_from_executable(bin/"cmctl", "completion")
```

Comment:
looks like completion cmd is broken, better to fix it.

Reply:
I'll take a look now.

Reply:
Haven't been able to check why the completion doesn't work, I think it's linked to the change in import paths

Reply:
looks like we need to enable it:

https://github.com/cert-manager/cmctl/blob/f5f4439e47f3092d4b6e05dd39ca4a115f8eef02/make/00_mod.mk#L24

Reply:
Fixed it, added the v2